floyd_warshall_algorithm/CPP/floyd_warshall_algorithm.cpp

```diff
@@ -0,0 +1,67 @@
+// Floyd-Warshall Algorithm: all pair shortest path
+#include <iostream>
+#include <climits>
+using namespace std;
+
+#define INF 999999999
+
+int main() {
+	freopen("input.txt", "r", stdin);
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*